5001-6000/LIT5354Mashaf.xml

```diff
@@ -62,19 +66,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          </langUsage>
       </profileDesc>
       <revisionDesc>
-         <change who="DE" when="2019-03-15">Created entity</change>         
+         <change who="DE" when="2019-03-15">created entity</change>
+         <change who="DN" when="2025-05-19">added incipit, bibliography, witnesses, work structure</change> 
+        
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
-         <div type="bibliography"/>
-         <!---->
-         <listRelation>
-            <relation name="dcterms:creator"
-                      active="LIT5354Mashaf"
-                      passive="PRS12520TaklaH"/>
+         <div type="bibliography">
+            <listBibl type="editions">
+               <bibl>
+                  <ptr target="bm:Platonov2002Lies"/>                  
+               </bibl>
+            </listBibl>            
+            <listRelation>    
+            <relation name="dcterms:creator" active="LIT5354Mashaf" passive="PRS12520TaklaH"/>               
          </listRelation>
+         </div>
+         <div type="edition" xml:lang="gez">
+            <note>The division into chapters is taken from
+               the edition <bibl><ptr target="bm:Platonov2002Lies"/></bibl>.</note>      
+            
+          <div type="edition">
```

Comment:
Sorry missed this. Do not nest div type edition in another div type edition.
1st level edition
2nd level textpart

Reply:
ok, corrected.